packages/@aws-cdk/aws-cloudwatch/test/dashboard.test.ts

```diff
@@ -161,6 +161,21 @@ describe('Dashboard', () => {
 
   });
 
+  test('DashboardName is exposed correctly when provided', () => {
```

Comment:
This tests is redundant,  If this member visibility change it will be caught by the API compatibility check.

I have marked the PR as no test required so it will pass the linter, lets remove this test